### DISCUSSION TITLE
`launch`ed processes are killed with xremap shutdown.  Is there a way to detach them?

### DISCUSSION THREAD

**Discussion by WJCFerguson** [paperwm/PaperWM#857] [Q&A]
When I use `launch` to spawn a long-running graphical program, restarting xremap kills them.

What's the right way to spawn long-running programs?  Do I just need to create an intermediate script?

**Comment by Lythenas** [comment]
I'm not familiar with xremap so you might need to ask that question in their repo. But by default on linux if you stop a process it will also stop all child processes.

**Comment by WJCFerguson** [comment]
Haha, sorry!  I got mixed up about where I was.
